src/components/carousel/index.js

```diff
@@ -0,0 +1,217 @@
+/* eslint-disable */
+import React, { Component } from 'react';
+import {
+  Carousel,
+  CarouselItem,
+  CarouselControl,
+  CarouselCaption
+} from 'reactstrap';
+
+const items = [
+  {
+    background: 'https://i0.wp.com/alliancegenome.files.wordpress.com/2016/11/banner1.jpg',
+    logo: 'https://i1.wp.com/alliancegenome.files.wordpress.com/2016/11/banner_1_flyin_logo.png',
+    captionHeader: 'The Gene Ontology Consortium and Six Model Organism Databases have joined together to form the Alliance of Genome Resources',
+    captionText: 'MORE INFORMATION',
+    link: '/about-us',
+  },
+  {
+    background: 'https://i1.wp.com/alliancegenome.files.wordpress.com/2016/11/banner2.jpg',
+    logo: '',
+    captionHeader: 'Have questions about the Alliance of Genome Resources?',
+    captionText: 'CONTACT US',
+    link: '/contact-us',
+  },
+  {
+    background: 'https://i2.wp.com/alliancegenome.files.wordpress.com/2016/11/banner3.jpg',
+    logo: '',
+    captionHeader: 'The Alliance Orthology Working Group is developing a gold standard set of orthologs to support comparative genomics across multiple model organisms.\n',
+    captionText: 'CLICK HERE FOR MORE INFO',
+    link: '/orthology',
+  }
+];
+
+class Homepage extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { activeIndex: 0 };
+    this.next = this.next.bind(this);
+    this.previous = this.previous.bind(this);
+    this.goToIndex = this.goToIndex.bind(this);
+    this.onExiting = this.onExiting.bind(this);
+    this.onExited = this.onExited.bind(this);
+  }
+
+  onExiting() {
+    this.animating = true;
+  }
+
+  onExited() {
+    this.animating = false;
+  }
+
+  next() {
+    if (this.animating) return;
+    const nextIndex = this.state.activeIndex === items.length - 1 ? 0 : this.state.activeIndex + 1;
+    this.setState({ activeIndex: nextIndex });
+  }
+
+  previous() {
+    if (this.animating) return;
+    const nextIndex = this.state.activeIndex === 0 ? items.length - 1 : this.state.activeIndex - 1;
+    this.setState({ activeIndex: nextIndex });
+  }
+
+  goToIndex(newIndex) {
+    if (this.animating) return;
+    this.setState({ activeIndex: newIndex });
+  }
+
+  render() {
+    const { activeIndex } = this.state;
+
+    const slides = items.map((item, idx) => {
+      return (
+        <CarouselItem
+          className="custom-tag"
+          key={idx}
+          onExiting={this.onExiting}
+          onExited={this.onExited}
+        >
+          <img className='logo-img' src={item.logo} />
+          <img className='slide-background' src={item.background} alt={item.altText} />
+
+          <a href={item.link}>
+            <CarouselCaption
+              className={item.className}
+              captionText={item.captionText}
+              captionHeader={item.captionHeader}
+            />
+          </a>
+
+        </CarouselItem>
+      );
+    });
+
+    return (
+      <div>
+      <style>
+        {
+            `.custom-tag {
```

Comment:
Does this way of declaring style potentially cause conflict? If so, maybe you could take the CSS module approach that's already widely used throughout this repo?